liquidpromptrc-dist

```diff
@@ -150,6 +150,15 @@ LP_RUNTIME_BELL_THRESHOLD=10
 # Recommended value is 1
 LP_ENABLE_VIRTUALENV=1
 
+# Display the containerization status for the current shell, if any.
+# Note that containers may inherit some, but not all variables from their
+# parent shell, so this may behave inconsisently with different container
+# software.  For example, some containers don't inherit anything unless
+# explicitly told to. Others get many things, but remove the
+# PROMPT_COMMAND that makes liquidprompt work.
```

Comment:
This is good documentation. I would hate for it to be lost in this example file. Move it to the `docs/config.rst` file instead.

As mentioned at the top of this file:
> This example config file does not contain all possible config options, nor does it have detailed option descriptions. See the documentation for that: [link]

Docs in this file are good, but they should be a short summary of what is found in the real documentation source.

Reply:
Should I just remove the value from the liquidpromptrc-dist entirely?

Reply:
I'd be fine with that, as it is just an example. I have tried to not form strong feelings on that subject so I don't need to ask every contributor to remove their changes from the file.

Reply:
In the latest version, I removed it from the `liquidpromptrc-dist`.